libsel4/sel4_arch_include/aarch64/sel4/sel4_arch/constants.h

```diff
@@ -259,7 +259,7 @@ SEL4_SIZE_SANITY(seL4_PUDEntryBits, seL4_PUDIndexBits, seL4_PUDBits);
 #else /*CONFIG_ARM_SMMU*/
 /* 1 slot at the end of the vspace is used to hold the VMID assigned to the vspace */
 /* 1 slot at the end of the vspace is used to hold the number of SMMU CBs assigned to the vspace */
-#define seL4_VSpaceReservedSlots 2
+#define seL4_VSpaceReservedSlots 2u
```

Comment:
Are you able to instead change the definiton on line 279 to:
`#define seL4_UserTop (0x000000ffffffffff - seL4_VSpaceReservedSlots * 0x40000000lu)`